features/admins/admin_post_issues.feature

```diff
@@ -19,14 +19,52 @@ Feature: Admin Actions to Post Known Issues
       And I follow "Show"
     Then I should see "First known problem"
 
-  Scenario: Edit known issues
+    Examples:
+    | role       |
+    | support    |
+    | superadmin |
+
+  Scenario Outline: Edit known issues
     Given I have posted known issues
+      And I am logged in as a "<role>" admin
     When I edit known issues
     Then I should see "Known issue was successfully updated"
       And I should not see "First known problem"
       And I should see "This is a bit of a problem, and this is too"
 
-  Scenario: Delete known issues
+    Examples:
+    | role       |
+    | support    |
+    | superadmin |
+
+  Scenario Outline: Delete known issues
     Given I have posted known issues
+      And I am logged in as a "<role>" admin
     When I delete known issues
     Then I should not see "First known problem"
+
+    Examples:
+    | role       |
+    | support    |
+    | superadmin |
+
+  Scenario Outline: Access is denied to unauthorized admins
```

Comment:
We're not really testing the access denial redirects here, so maybe like "Links for editing and creating known issues are not present for unauthorized admins"?